`npm install` fails on fresh clone

When I clone the repo and try to run `npm install`, I get the following error:

```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /Users/nicholas/projects/y-sweet/node_modules/yjs/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/Users/nicholas/projects/y-sweet/node_modules/yjs/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nicholas/.npm/_logs/2024-03-01T02_20_25_631Z-debug-0.log
```

This is because the `tests/` module imports `yjs` with:

```
"yjs": "file:../node_modules/yjs"
```

Which doesn't work if you don't have `node_modules`, so you end up in a sorta chicken and the egg situation where you need `node_modules` to `npm install` but you create `node_modules` by running `npm install`.

As a workaround I just changed the dependency specification back to a version, but idk if that'll work as a permanent fix

--- a/tests/package.json
+++ b/tests/package.json
@@ -14,7 +14,7 @@
     "typescript": "^5.1.6",
     "vitest": "^0.34.4",
     "ws": "^8.13.0",
-    "yjs": "file:../node_modules/yjs"
+    "yjs": "^13.6.7"
   },
   "dependencies": {
     "@y-sweet/react": "0.1.0",

